lib/vra/client.rb

```diff
@@ -93,19 +93,17 @@ def authorized?
       return false if @bearer_token.value.nil?
 
       response = http_head("/identity/api/tokens/#{@bearer_token.value}", :skip_auth)
-      if response.code == 204
-        true
-      else
-        false
-      end
+      response.success?
```

Comment:
Do we know if a 200..203 status code is OK for this call? In our testing and the API docs, it says to expect a 204. We may cause unexpected issues by just assuming any 2xx status code is OK for this operation.